lib/manager/travis/readme.md

```diff
@@ -1 +1,28 @@
-This manager is intended to keep Travis config files (`.travis.yml`) up-to-date. Currently it manages only the `node_js` section of files only.
+This manager is intended to keep Travis config files (`.travis.yml`) up-to-date.
+Currently it manages only the `node_js` section of files only.
+
+An important limitation to note is that Renovate does not currently "understand" Travis's node matrix concept, so it will try to update all found node versions to the latest LTS, e.g.
```

Comment:
```suggestion
An important limitation is that Renovate does not currently "understand" Travis's node matrix concept, so it will try to update all found node versions to the latest LTS, e.g.
```

I think we can simplify here. 😄